src/read.c

```diff
@@ -760,6 +760,10 @@ static avifMeta * avifMetaCreate()
 
 static void avifMetaDestroy(avifMeta * meta)
 {
+    if (meta == NULL) {
```

Comment:
The commit message should also acknowledge the person who ran nallocfuzz and reported this bug to us (Philippe Antoine).

Reply:
> The commit message should also acknowledge the person who ran nallocfuzz and reported this bug to us (Philippe Antoine).

> Found by Nallocfuzz (https://github.com/catenacyber/nallocfuzz).

This was the format I used in other projects, that was suggested by Philippe. I can add his name though.